src/helpers/whitelist.ts

```diff
@@ -31,12 +31,18 @@ const getParsedAddressList = (address: string | string[]) => {
 };
 
 export class Whitelist {
-  static async checkUrl(url: string | undefined): Promise<boolean> {
+  static async checkUrl(
+    url: string | undefined,
+    forceSkip = true,
+  ): Promise<boolean> {
```

Comment:
Looks like we don't need async here

Reply:
Init remote config if it's not inited yet
   ```ts
    if (!RemoteConfig.isInited) {
      await RemoteConfig.init();
    }
```